canonical/charmhub.io
# update charmhub beta

## Done
- update beta app
## How to QA
- Go to https://charmhub-io-1670.demos.haus/beta/store.json, https://charmhub-io-1670.demos.haus/beta/kafka/card.json, https://charmhub-io-1670.demos.haus/beta/charms and ensure it all works as expected
## Issue / Card
Fixes #

## Screenshots
